sdk/modelsrepository/Azure.Iot.ModelsRepository/README.md

```diff
@@ -0,0 +1,89 @@
+# Azure IoT Models Repository client library for .NET
+
+This library provides functionality for interacting with the Azure IoT Models Repository. It also aims to provide a consistent experience working with digital twin model repositories following Azure IoT conventions.
```

Comment:
it might be useful you include a link to the models repository

Reply:
done